server/graphql/utils/request.ts

```diff
@@ -20,19 +20,68 @@ const getRequestsById = (requestIds, dataSources) => {
   return requestIds.map(id => dataSources.requests.getById(id))
 }
 
-const updateRequestHelper = (request, dataSources): Promise<ServerResponseInterface> => {
+const updateRequestHelper = (request, dataSources): Promise<Document> => {
   if(!request.id) {
     throw new UserInputError('Missing argument value', { argumentName: 'id' })
   }
-  const requestTypeId = dataSources.requests.getById(request.id.toString()).requestType
-  return dataSources.requests.update(request)
-    .then(res => {
-      updateRequestTypeHelper({"id": requestTypeId}, dataSources)
-      return res
-    })
-    .catch(error => {
-      return error
-    })
+
+  const currentRequestCopy = dataSources.requests.getById(request.id.toString()).toObject()
+  currentRequestCopy.id = currentRequestCopy._id
+  const currentRequest = dataSources.requests.getById(request.id.toString())
```

Comment:
To reduce searches through the cache, we could try
```
const currentRequest = dataSources.requests.getById(request.id.toString())
const currentRequestCopy = currentRequest.toObject()
currentRequestCopy.id = currentRequestCopy._id
```

Reply:
Updated.